src/index.ts

```diff
@@ -7,16 +9,20 @@ import parseSpiResult from "./use-cases/parseSpiResult"
 import transformSpiToAho from "./use-cases/transformSpiToAho"
 
 export default (message: string, pncGateway: PncGateway): BichardResultType => {
+  let hearingOutcome: AnnotatedHearingOutcome
```

Comment:
nit: we can declare this `const` on line 15, it's value doesn't (and shouldn't) change after assigning

Reply:
Totally agree. However this is an artefact of the `parsing` side of this ticket that didn't get cleaned up when they split. Going to leave it for now, since it's just about to be changed to this anyway